drivers/enc28j60/enc28j60.c

```diff
@@ -248,10 +248,27 @@ static int nd_send(netdev_t *netdev, const iolist_t *iolist)
 
     mutex_lock(&dev->devlock);
 
-#ifdef MODULE_NETSTATS_L2
-    netdev->stats.tx_bytes += count;
-#endif
+    if (cmd_rcr(dev, REG_ECON1, -1) & ECON1_TXRTS)
+    {
+        /* there is already a transmission in progress */
+        if (xtimer_now_usec() - dev->tx_time > 2429)
+        {
+            /* if transmission time exceeds 2.4287 ms, we suppose that
```

Comment:
ah here is some docu, still would prefer to use macro/constant

Reply:
Sure, I can introduce a macro/constant definition.